src/nls/it/strings.js

```diff
@@ -68,8 +68,8 @@ define({
     "ERROR_DELETING_FILE_TITLE"         : "Errore durante il tentativo di eliminazione del file",
     "ERROR_DELETING_FILE"               : "Errore durante il tentativo di eliminazione del file <span class='dialog-filename'>{0}</span>. {1}",
     "INVALID_FILENAME_TITLE"            : "Il nome del file non è valido",
-    "INVALID_FILENAME_MESSAGE"          : "Il nome del file non può contenere i seguenti caratteri: {0}",
-    "FILE_ALREADY_EXISTS"               : "Il file <span class='dialog-filename'>{0}</span> esiste già.",
+    "INVALID_FILENAME_MESSAGE"          : "Il nome {0} non può usare caratteri di sistema riservati, finire con un punto (.) o contenere i seguenti caratteri: <code class='emphasized'>{1}</code>",
```

Comment:
![image](https://cloud.githubusercontent.com/assets/2641501/7662893/bc07890a-fb62-11e4-92ac-2bbe8a91e527.png)
I don't think that's the expected outcome. (`{0}` is replaced with either `Nomi dei file` or `Nomi cartelle`).

If you can't make it work with the strings available (our nls resources are sadly still very English-oriented and thus, it's not always easy to make translations work), try something like `Name of file/folder` translated to Italian.